packages/kit/src/runtime/load.js

```diff
@@ -52,5 +54,12 @@ export function normalize(loaded) {
 		}
 	}
 
+	if (dev && /** @type {any} */ (loaded).context) {
```

Comment:
Added the warning. Using `dev` from `./app/env` here, which wasn't used anywhere else yet (only for user-facing-code), hope that doesn't have any bad implications.

Reply:
Well, it does apperently, tests are failing now because `env` is not set in tests. What now?

Reply:
Would `context` still work in prod? If not I think we should make a hard error here.

But regarding the error, I believe it's because `import.meta.env` is non-standard in nodejs so it would resolve to `undefined`, hence the error. A workaround is to shim it with `import.meta.env = { /* ... */ }` at the top of this file, though we need to make sure we do that for tests only. Using `process.env.NODE_ENV` may be fine for this as well.

Reply:
I'd expect it to be defined in the integration tests. I think the issue is around this being unit tested. Rather than modify this to shim it, I wonder if we could move the check to another location that's called in integration tests, but not unit tests.

Reply:
This sounds wrong to me. I picked this method as it seems the central location to validating the load response. Also we probably don't have this check in for long, we could remove it at 1.0 and before that shimming it sounds like a pragmatic solution in the meantime

Reply:
Ok. That makes sense